src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/RoleAssignmentApi.java

```diff
@@ -7,7 +7,9 @@
 import org.springframework.web.bind.annotation.PostMapping;
```

Comment:
Do we have unit test coverage for the changes introduced?

Reply:
We don't cover other Api clients directly but as part of the junit test for the class that uses the API. This one, in particular, is covered by this test RoleAssignmentServiceTest